### PR TITLE
user api endpoints to view create and delete users

### PR DESCRIPTION
**Pull request by harsh-ux**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -26,10 +26,6 @@ Procedure:
     ```
     pipenv shell
     ```
-- Change to source code directory
```
Comment (manav1403) [inline backend/README.md:29]: why this change is readme

Comment (harsh-ux) [reply backend/README.md:29]: I havent made this chane must be some error

File: backend/config/urls.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
     path('api/team/',include("team.urls")),
     path('api/auth/', include("apiauth.urls")),
     path('api/projects/', include("projects.urls")),
+    path('api/user/', include("apiauth.urls")),
```
Comment (manav1403) [inline backend/config/urls.py:25]: duplicate route with api/auth is creating warning
remove api/users
